Genpop packed starlighted

Makeshiftt opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
Starlights the Upstream Packed! This includes everything that is required, plus some of the recent furniture additions sprinkled around where they make sense. Also includes an updated Supermatter similar to Leth's, that should be complete enough to enable the Supermatter to be a primary engine more consistently, if engineering so chooses.
<!-- What do you propose to change with your PR? -->

## Why we need to add this
More map variety is always nice! Especially in the lowpop pool.
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->

## Media (Video/Screenshots)
<img width="4800" height="3776" alt="image" src="https://github.com/user-attachments/assets/dff2150e-c91b-492e-9ff3-105fb2bd7221" />
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: Makeshift
- add: Reintroduced Packed to the map pool, SM included.
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->

Makeshiftt committed:
Starlighted the Upstream Packed.

Makeshiftt committed:
Fixes a couple invalid references.

Makeshiftt committed:
Merge branch 'ss14Starlight:starlight-dev' into genpopPackedStarlighted

PubliclyExecutedPig reviewed (approved):
changes look good to me!

Conflee commented:
Salv/Mining needs one of the venders that gives picks and grappling hooks.

Makeshiftt committed:
Adds a salvage vendor.

Makeshiftt commented:
Whoops, added it back o7

Happyrobot33 committed:
Merge branch 'starlight-dev' into genpopPackedStarlighted

CawsForConcern commented:
Some comments for stuff that should be added:
- Cell Recharger in Chemistry
- Cell Recharger in Brigmedic
- Document Printer in HOP
- Nanomed Civilian vending machine(s). I usually do one around Medical and/or one around Evac.

Everything else looks pretty great! <3

Makeshiftt committed:
Added a couple Nanomed Civilian/Bandaids
Added a printer to HoPs room
Added a cell recharger next to chem
Properly linked some added firelocks

Makeshiftt commented:
Done! (There was already a cell recharger in brigmed!)

Makeshiftt committed:
Adds a Salvage Lead, with locker, spawn and job slot.

Happyrobot33 committed:
Merge branch 'starlight-dev' into genpopPackedStarlighted
